inspirehep/modules/theme/static/scss/format/brief-results.scss

```diff
@@ -508,4 +508,17 @@ i.fa-chevron-right.facet-slider {
 
 .brief-results-sortby {
 	color: #737373;
+}
+
+div > .modal-dialog {
+    width: 900px;
```

Comment:
This should be tested in all screens. Instead of absolute pixels, it should be a percentage so that it displays correctly in all viewports.